### PR TITLE
Make language sign more recognizable

### PR DESCRIPTION
**Pull request by alalav1**
Add current language code next to the symbol and use a larger button with borders.

### CODE REVIEW

**Comment by Plustest** [comment]
Can one of the admins verify this patch?

File: course/templates/course/_course_menu.html [hunk 1]
```diff
@@ -10,13 +10,17 @@
...
     <div class="col-sm">
-			<div class="dropdown h4">
-				<input type="image" src="{% static 'images/uls.svg' %}" alt="{% trans 'Language' %}"
-				class="btn btn-xs" id="lang-dropdown" type="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false" />
+			<div class="dropdown">
+				{% get_current_language as LANGUAGE_CODE %}
+				<button class="btn btn-default dropdown-toggle" type="button" id="lang-dropdown"
+				data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+					<span style="display: inline-block; background-image: url({% static 'images/uls.svg' %}); width: 20px; height: 20px;"></span>
+					<span style="font-size: 18px;">{{ LANGUAGE_CODE }}</span>
```
Comment (raphendyr) [inline course/templates/course/_course_menu.html]: Would it be nicer to add `|title` filter to capitalize the language code?

**Comment by raphendyr** [comment]
Can you include a screenshot of the language button area, thanks.

**Comment by raphendyr** [comment]
add to whitelist

**Comment by Mankro** [comment]
The language button has already been made look more like a button in the master branch during the summer 2020, but this PR still includes the idea of adding the textual language code to the button.

**Pull request closed without merging**